.github/workflows/test_manylinux.yml

```diff
@@ -48,6 +48,16 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
+      - name: Downgrade PIP
+        run: |
+          # pip v22+ will fail on error rather than falling back to the
+          # previous release
+          # the old behaviour is better for this check
+          if (( $(pip --version | sed 's/pip \([0-9]*\).*/\1/') >= 22 ))
+          then
+            pip install 'pip<22'
+          fi
+
```

Comment:
Can we not do `pip install --prefer-binaries` instead?

Reply:
Not sure how that would help?

Reply:
It might lead to pip trying to install the compatible wheel instead of the tar.gz that it's been failing with. I haven't tried it though, whatever